### DISCUSSION TITLE
멀티플레이 관련 질문이 있습니다 :)

### DISCUSSION THREAD

**Discussion by kimyang-sun** [naverz/zepeto-studio-kor#913] [5. Q&A]
안녕하세요! :)
궁금한점이 있습니다!

유니티 제페토 스크립트로 만든 월드가 심사를 거쳐 배포가 되면
기본적으로 플레이어들이 같이 접속해서 서로 채팅하거나 하는 기능은 적용이 되어있는 상태인걸까요?

아니면 올려주신 멀티플레이 예제 [https://github.com/naverz/zepeto-multiplay-example](url)
를 참고해서 멀티플레이 부분도 직접 작업해서 넣어줘야 같이 접속해서 멀티플레이를 즐길 수 있는걸까요??

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 멀티플레이 요소를 추가해야 멀티플레이 월드가 됩니다.
기본적으로는 싱글플레이 월드입니다. 
감사합니다.

**Reply by kimyang-sun** [reply]
답변 감사합니다 :)
